src/index.ts

```diff
@@ -96,7 +100,7 @@ export class Probot {
     process.on('unhandledRejection', this.errorHandler)
 
     // Load the given apps along with the default apps
-    apps.concat(defaultApps).forEach(app => this.load(app))
+    apps.concat(defaultApps).forEach((app: any) => this.load(app))
```

Comment:
> any

Could this be `ApplicationFunction`?

Reply:
Yep! Missed that one.

Reply:
It was inferring correctly as `string | ApplicationFunction`. I realize I added this while testing something else, so I removed it again in 1dd70c3bc2844448c672779e3d5dfae53d5d0227